CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGE LOG
 
+## [0.9.2] 
```

Comment:
haha uhh so we still haven't been able to get our release out, so we're still on 0.9.1